tools/toolchain/webasm-darwin/cc_toolchain_config.bzl

```diff
@@ -0,0 +1,182 @@
+load(
```

Comment:
ooc, what are these two new large wasm build files for? did you copy them from somewhere?

Reply:
They're the new way to describe a C++ toolchain to bazel. This is also why the CROSSTOOL files are removed.

Reply:
I should also point out that I generated them by using a tool from rules_cc that converts from the old CROSSTOOL format.